### PR TITLE
Release v1.4.0

### PR DESCRIPTION
**Pull request by longinoa**

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Is this title v1.4.0 ?

**Comment by longinoa** [comment]
Thanks @KazuCocoa! copypasta error

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+_2016_09_07
+
+ * **Add UI Accessibility Properties to Styles tab**
+   Added support for accessibility inspection, which allows users to select
+   a View and see whether or not it will be focusable by an Accessibility
+   Service, why it will or won't be focusable, the text description sent to
+   Accessibility Services, and any AccessibilityActions that are currently
+   available on the View.
+
+ * Fix #333: Remove duplicates in Elements tree
```
Comment (rickbrew) [inline CHANGELOG.md]: this was already included in `1.3.1`

**Comment by rickbrew** [comment]
Looks good! :+1:

**Pull request merged**